### DISCUSSION TITLE
设置只允许管理员上传，评论区变成我的图片上传的图床。

### DISCUSSION THREAD

**Discussion by xiamuguizhi** [ArtalkJS/Artalk#130] [Show and tell]
别说还很好用，粘贴上传，拖拽上传，对于我来说就是自己图床 哈哈！

![image](https://user-images.githubusercontent.com/19650297/165317009-dbdae653-1395-42da-b989-d7ed7e80ea04.png)

**Comment by qwqcode** [comment]
哈哈，可以的！
